STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -153,7 +153,7 @@ The protocol assumes the following preconditions are met:
   management interface;
 - The NDC has registered an ACME account with the IdO;
 - NDC and IdO have agreed on a "CSR template" to use, including at a minimum:
-  subject name (e.g., "somesite.example.com"), requested algorithms and key
+  subject name (e.g., `somesite.example.com`), requested algorithms and key
```

Comment:
I honestly don't understand why you spent time on the double quotes :-)

Reply:
Just to please the xml2rfc HTML renderer.  It was a cheap, mechanical `:'<,'>:s/"\([^"]*\)"/`\1`/g` in vim, so it didn't take more than a few seconds.  That said, I should have sent is as a separate PR.